lighthouse-core/test/audits/audit-test.js

```diff
@@ -99,4 +99,77 @@ describe('Audit', () => {
     assert.equal(result.score, null);
     assert.equal(result.scoreDisplayMode, 'error');
   });
+
+  // is audit the right place to put this logic?
```

Comment:
not really, but it's where we have it right now so I think you're fine :)

might make sense to split off into some `details-helpers` file or something in the future